css/styles.css

```
@@ -16417,3 +16417,319 @@ input[type="number"] {
 }
+
+/* =============================================================================
+   RATIOS PANEL (STRK-270) — shared Layout C component, tokens only (D-7).
+   Lifted from playground/ratios-panel-playground.html. Rendered by the
+   in-app modal host (STRK-271) and the standalone /ratios/ page (STRK-273).
+   Accent follows the DENOMINATOR metal — the thing being priced against —
+   matching the per-card accent convention the spot chips already use.
+   ============================================================================= */
+.gsr-panel {
+  --gsr-accent: var(--gold);
+  padding: 16px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)